locomotion-sdk/src/services/ride-history.js

```diff
@@ -0,0 +1,11 @@
+import network from './network';
+
+const RideHitory = {
+  getHistory: async () => {
+    const req = await network.get('api/v1/me/rides/history');
+    const { data: res } = req;
+    return res || {};
```

Comment:
you can do it in easier way:
`
const { data } = await network.get('api/v1/me/rides/history');
return data;
`